cmd/spire-agent/cli/run/run.go

```diff
@@ -229,6 +234,61 @@ func mergeInput(fileInput *Config, cliInput *agentConfig) (*Config, error) {
 	return c, nil
 }
 
+func setupTrustBundle(ac *agent.Config, c *Config) error {
+	// If trust_bundle_url is set, download the trust bundle using HTTP and parse it from memory
+	// If trust_bundle_path is set, parse the trust bundle file on disk
+	// Both cannot be set
+	// Unless insecure_bootstrap is set, the trust bundle URL must start with HTTPS
+	//    (if it is set, then downloading the trust bundle just emits messages in the logs,
```

Comment:
I think one of the proposed ways of handling this specific configuration was to _try_ and authenticate the server using the downloaded bundle, but otherwise just log and carry on if that failed.

I don't really see what value is added in allowing a configuration where the bundle is downloaded but not used.

Reply:
Let's just require it to always start with https. If anyone can come up with a good reason not to, it's easy enough to change :)